chess/__init__.py

```diff
@@ -626,14 +626,15 @@ def __init__(self, from_square, to_square, promotion=None, drop=None):
         self.promotion = promotion
         self.drop = drop
 
-    def uci(self):
+    def pacn(self):
         """
-        Gets an UCI string for the move.
+        Gets an PACN string for the move.
```

Comment:
@niklasf do you have another suggestion for this? xboard and uci both use this coordinate notation for moves so calling it uci seems misleading...but there is no standard name for it either.